src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background-color: white;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.link {
+  display: inline-block;
+  line-height: 60px;
+  color: black;
+  background-color: white;
+  text-decoration: none;
+  font-style: normal;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list-item :hover {
+  text-decoration: none;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  text-decoration: none;
```

Comment:
Try not to duplicate this property `text-decoration: none;`, but reset it for the tag a

Reply:
Okay. thank you for your advice)